pkgs/applications/misc/kjv/default.nix

```
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, gawk, gnutar, gnumake, coreutils, gzip }:
+
+stdenv.mkDerivation rec {
+  name = "kjv";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "LukeSmithxyz";
+    repo = "${name}";
+    rev = "f6e6f639b135414659179b09a5fdec4ad8add40e";
+    sha256 = "1yn91v8rsfdsd8p1bfx3v54cvyzwg5mka3inzdkc60k8dahfq8g5";
+  };
+
+  builder = ./builder.sh;
```

You should be able to use the Makefile they provide, just pass it the right makeFlags
```suggestion
  makeFlags = [
    "PREFIX=$(out)"
  ];
```
should be able to remove your custom builder.sh :)